src/signature.ts

```diff
@@ -52,9 +52,9 @@ export class Signature {
     return this.value;
   }
 
-  public verifyAggregate(publicKey: Uint8Array[], message: Uint8Array): boolean {
+  public verifyAggregate(publicKeys: PublicKey[], message: Uint8Array): boolean {
```

Comment:
Would it make sense to expose both versions with different names? Does the variant taking compressed keys have no value?

Reply:
you have verifyAggregate in root that takes compressed stuff